code/modules/clothing/clothing.dm

```diff
@@ -120,6 +120,23 @@
 	icon_state = "earmuffs"
 	item_state = "earmuffs"
 	flags_equip_slot = SLOT_EAR
+	inherent_traits = list(TRAIT_ITEM_EAR_EXCLUSIVE)
+	clothing_traits = list(TRAIT_SCREECH_IMMUNE)
+	time_to_equip = 40
+
+/obj/item/clothing/ears/earmuffs/equipped(mob/user, slot)
+	. = ..()
+	if(slot == (WEAR_L_EAR||WEAR_R_EAR))
+		user.client.soundOutput.status_flags |= EAR_DEAF_MUTE
```

Comment:
This can't be only an equipped() thing or you can just relaunch the game, get a fresh client without mute, and voilà

This also runtime errors if you equip earmuffs to a SSD/unowned mob